store/project_member.go

```diff
@@ -101,6 +101,26 @@ func (s *Store) GetProjectMemberByID(ctx context.Context, id int) (*api.ProjectM
 	return projectMember, nil
 }
 
+// GetProjectMemberByProjectIDAndPrincipalID gets an instance of ProjectMember by projectID and principalID.
+func (s *Store) GetProjectMemberByProjectIDAndPrincipalID(ctx context.Context, projectID int, principalID int) (*api.ProjectMember, error) {
```

Comment:
Prefer to add a general function `GetProjectMember(ctx context.Context, find *api.ProjectMemberFind)`